type/__apt_source/parameter/optional

```diff
@@ -1,4 +1,5 @@
 state
 distribution
 component
-arch
\ No newline at end of file
+arch
+signed-by
```

Comment:
The [man page](https://manpages.debian.org/bullseye/apt/sources.list.5.en.html#THE_DEB_AND_DEB-SRC_TYPES:_OPTIONS) describes this option as being a (comma separated) list.

So I think this should be a `optional_multiple` option.